charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -13,6 +13,149 @@ Those components are needed to have the full experience on NROne kubernetes expl
+
+{{- end }}
+
+{{- if .Values.nodeAffinity }}
+
+###########
+# WARNING #
+###########
+
+You have specified "nodeAffinity" into values, however the new version of the chart
+support different nodeAffinity options per component, therefore the value specified has been ignored.
+
+Please use ksm.affinity.nodeAffinity, controlPlane.affinity.nodeAffinity and kubelet.affinity.nodeAffinity.
+
+{{- end }}
+
+{{- if .Values.resources }}
+
+###########
+# WARNING #
+###########
+
+You have specified the "resources" into values, however the new version of the chart
+support different resources per component, therefore the value specified has been ignored.
+
+Please use ksm.resources, controlPlane.resources and kubelet.resources.
+
+{{- end }}
+
+{{- if .Values.tolerations }}
+
+###########
+# WARNING #
+###########
+
+You have specified "tolerations" into values, however the new version of the chart
+support different tolerations per component, therefore the value specified has been ignored.
+
+Please use ksm.tolerations, controlPlane.tolerations and kubelet.tolerations.
+
+{{- end }}
+
+{{- if ( or image.repository image.tag image.pullPolicy image.pullSecrets) }}
+
+###########
+# WARNING #
+###########
+
+You have specified into values options "image.*".
+The following values are no longer supported and are currently ignored.
+ - image.repository
+ - image.tag
+ - image.pullPolicy
+ - image.pullSecrets
+
+Notice that the 3.x version of the integration uses 3 different images.
+Please set:
+ - images.forwarder.* to configure the image in charge of sending data to newrelic backend
+ - images.agent.* to configure the image bundling the agent and onHost integration
+ - images.integration.* to configure the image in charge of scraping k8s data
+
+{{- end }}
+
+{{- if (or .Values.enableWindows .windowsOsList .windowsSecurityContext .windowsNodeSelector)}}
+
+###########
+# WARNING #
+###########
+
+Windows is currently supported by 2.x charts only and "windowsOsList" "windowsSecurityContext" "windowsNodeSelector"
+are no longer supported.
+
+{{- end }}
+
+{{- if ( or .Values.etcdTlsSecretName .etcdTlsSecretNamespace )}}
+
+###########
+# WARNING #
+###########
+
+Values "etcdTlsSecretName" and "etcdTlsSecretNamespace" are no longer supported, please specify them in the config
+file. Example:
+# - endpoints:
+#     - url: https://localhost:9979
+#       insecureSkipVerify: true
+#       auth:
+#         type: mTLS
+#         mtls:
+#           secretName: secret-name
+#           secretNamespace: secret-namespace
+
+{{- end }}
+
+{{- if .Values.logFile }}
+
+###########
+# WARNING #
+###########
+
+The "logFile" value is no longer supported, please specify the option "log_file" directly in "common.agentConfig".
+
+{{- end }}
+
+{{- if .Values.runAsUser }}
+
+###########
+# WARNING #
+###########
+
+The "runAsUser" value is no longer supported, please specify it with the value "securityContext.runAsUser".
+
+{{- end }}
```

Comment:
Seems like this should be shim-able with ease, is there any reason to leave it out?

Reply:
mainly time, added it